samples/go/csv/read.go

```diff
@@ -146,17 +131,18 @@ func ReadToMap(r *csv.Reader, headersRaw []string, pkIdx int, kinds KindSlice, v
 
 		fieldIndex := 0
 		var pk types.Value
-		fields := make(types.ValueSlice, len(headers))
+		fields := make(types.ValueSlice, len(headersRaw))
 		for x, v := range row {
-			if x == pkIdx {
-				pk, err = StringToValue(v, pkKind)
-			} else if fieldIndex < len(headers) {
+			if fieldIndex < len(headersRaw) {
```

Comment:
It looks like we're ignoring any fields that exceed the number of headers. Shouldn't we print a warning or something? Also, I don't see why we need the fieldIndex var here. Can't you just use 'x'?